### PR TITLE
feature: add login layout

### LINKED ISSUE #4: feature: add login page

### PR DESCRIPTION
**Pull request by NabilSli**
fix #4

### CODE REVIEW

**Review by tony-go — APPROVED** [review]

File: FrontEnd/assets/style.css [hunk 1]
```diff
@@ -52,11 +52,14 @@ body {
 	margin:auto;
 	font-family: 'Work Sans' ;
 	font-size: 14px;
+	background-color: #FFFEF8;
```
Comment (tony-go) [inline FrontEnd/assets/style.css:55]: Each time you add a new color, maybe could you create a variable or at least add a todo item?

File: FrontEnd/assets/style.css [hunk 2]
```diff
@@ -178,8 +184,48 @@ input[type="submit"]{
...
+}
+.loginButton {
+	margin: 0;
+}
+.loginButton input {
+	width: 180px;
+	height: 36px;
+}
+#forgottenPassword {
+	color: #000000;
```
Comment (tony-go) [inline FrontEnd/assets/style.css:230]: ditto

**Review by tony-go — APPROVED** [review]
nits but lgtm otherwise

File: FrontEnd/assets/style.css (hunk 1 above)
Comment (NabilSli) [reply FrontEnd/assets/style.css:55]: ok i'll do it for good practises

File: FrontEnd/assets/style.css (hunk 2 above)
Comment (NabilSli) [reply FrontEnd/assets/style.css:230]: 👌

**Pull request merged**